assets/js/components/user-input/UserInputQuestionnaire.js

```diff
@@ -42,13 +43,16 @@ import {
 } from './util/constants';
 import useQueryArg from '../../hooks/useQueryArg';
 import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
-const { useSelect } = Data;
+import { STORE_NAME as CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+const { useSelect, useDispatch } = Data;
 
 export default function UserInputQuestionnaire() {
 	const steps = [ ...USER_INPUT_QUESTIONS_LIST, 'preview' ];
 
 	const [ activeSlug, setActiveSlug ] = useQueryArg( 'question', steps[ 0 ] );
 	const [ redirectURL ] = useQueryArg( 'redirect_url' );
+	// The single variable is used to edit *just one question* from the summary screens on the user input page and the settings page.
+	const [ single, setSingle ] = useQueryArg( 'single', false );
```

Comment:
@benbowler let's move this hook to the `UserInputQuestionWrapper` component to escape a need to pass it via props. Just destruct the first value only in the wrapper component: `const [ single ] = useQueryArg( ... )`